app/code/Magento/Newsletter/Model/Config.php

```diff
@@ -41,7 +41,7 @@ public function __construct(
      * @param string $scopeType
      * @return bool
      */
-    public function isActive(string $scopeType = ScopeConfigInterface::SCOPE_TYPE_DEFAULT): bool
+    public function isActive(string $scopeType = \Magento\Store\Model\ScopeInterface::SCOPE_STORE): bool
```

Comment:
I believe we shouldn't change the scope here, as you can see the scope is a parameter and can be requested differently by who is calling it, I would recommend changing `\Magento\Customer\Block\Form\Register::isNewsletterEnabled` to request the store scope instead.

Reply:
ok, I'll work on it